### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vldtsw**

### CODE REVIEW

**Review by denyslipin — APPROVED** [review]

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+    <div class="container-fluid">
+        {% block sidebar %}{% endblock %}
+        {% block content %}{% endblock %}
+        {% block pagination %}{% endblock %}
+    </div>
+
+</body>
+</html>
```
Comment (IvanRamyk) [inline templates/base.html:21]: Please, add a blank line at the end of the file

File: templates/taxi/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+{% block content %}
+    <h1>Taxi Service</h1>
+    <p>Welcome to great Taxi Service!</p>
+    <h3>Taxi content:</h3>
+    <ul>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_manufacturers }}</li>
+        <li>Number of cars: {{ num_cars }}</li>
+    </ul>
+{% endblock %}
```
Comment (IvanRamyk) [inline templates/taxi/index.html:12]: Please, add a blank line at the end of the file

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
Please, add some styles to `style.css` file to check it works properly

File: templates/taxi/index.html (hunk 2 above)
Comment (vldtsw) [reply templates/taxi/index.html]: Fixed. I thought it was not necessary because it is not a python file

File: taxi/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from taxi.views import index
+
+urlpatterns = [
+    path("", index),
```
Comment (MasterpieceElbow) [inline taxi/urls.py:6]: Add a name for the path.

File: taxi_service/settings.py [hunk 4]
```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [BASE_DIR, "templates"],
```
Comment (MasterpieceElbow) [inline taxi_service/settings.py:59]: You added two dirs instead of adding the correct path.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by MasterpieceElbow — APPROVED** [review]

**Pull request closed without merging**